chatbot/routes.py

```diff
@@ -109,6 +130,17 @@ def read_users_me(current_user: User = Depends(get_current_user)):
     return current_user
 
 
+@router.get("/models")
+def list_models():
+    gpt_models = [v.value for k, v in GPTModelEnum.__members__.items()]
```

Comment:
Você pode itera sobre os valores assim:

`gpt_models = [v.value for k, v in GPTModelEnum]`